influxdb_client/client/tasks_api.py

```diff
@@ -11,6 +11,23 @@
     AddResourceMemberRequestBody, RunManually, Run, LogEvent
 
 
+class _TasksIterator:
+    def __init__(self, values, next) -> None:
+        self.values = values
+        self.next = next
+
+    def __iter__(self):
+        return self if self.values else (_ for _ in ())
+
+    def __next__(self):
+        if not self.values:
+            if self.next:
+                self.values, self.next = self.next()
+            if not self.values:
+                raise StopIteration
+        return self.values.pop(0)
```

Comment:
Please refactor the iterator to a generic format. We should be able to reuse `_TasksIterator` (as `_NextIterator`) in other APIs, such as `organizations_api`.